### PR TITLE
Updating genpolicy to accomdate CRI request format

### PR DESCRIPTION
**Pull request by ksayid**
### Environment Variables Update
Previous format:
```json
"Env": [
  "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
]
```
New format:
```json
"envs": [
  {
    "key": "PATH",
    "value": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
  }
]
```
### Removed Features:
* Removed dmverity hashing
* Removed references to pause container

### CODE REVIEW

File: src/tools/genpolicy/src/containerd.rs [hunk 1]
```diff
@@ -30,14 +30,16 @@ pub fn get_process(privileged_container: bool, common: &policy::CommonData) -> p
         User: Default::default(),
         Args: Vec::new(),
         Env: Vec::new(),
```
Comment (SethHollandsworth) [inline src/tools/genpolicy/src/containerd.rs]: let's comment out `Env` in the output/wherever it's no longer needed

File: src/tools/genpolicy/src/registry.rs [hunk 2]
```diff
@@ -264,207 +267,210 @@ async fn get_image_layers(
...
-    if error {
-        // remove the cache file if we're using it
-        if use_cached_files {
-            std::fs::remove_file(cache_file)?;
-        }
-        warn!("{error_message}");
-    }
-    Ok(verity_hash)
-}
+// async fn get_verity_hash(
```
Comment (SethHollandsworth) [inline src/tools/genpolicy/src/registry.rs:281]: if these functions aren't referenced anywhere, I think it's fine to leave them uncommented. Unless they're throwing compilation errors

File: src/tools/genpolicy/src/verity.rs [hunk 3]
```diff
@@ -1,222 +1,224 @@
...
-
-pub fn no_write(_: &mut File, _: &[u8], _: u64) -> io::Result<()> {
-    Ok(())
-}
+// // Copyright (c) 2023 Microsoft Corporation
+// //
+// // SPDX-License-Identifier: Apache-2.0
+// //
+
+// // use generic_array::{typenum::Unsigned, GenericArray};
```
Comment (SethHollandsworth) [inline src/tools/genpolicy/src/verity.rs:6]: same thing as above with unreferenced code

**Review by SethHollandsworth — CHANGES_REQUESTED** [review]
just a couple things to look at, thanks!

File: src/tools/genpolicy/src/registry.rs (hunk 2 above)
Comment (ksayid) [reply src/tools/genpolicy/src/registry.rs:281]: yeah they throw compilation errors, but I think there's a flag for ignoring these type of errors

Comment (SethHollandsworth) [reply src/tools/genpolicy/src/registry.rs:281]: ahh leave them commented then, we'll delete once we're sure they're not in use

File: src/tools/genpolicy/src/containerd.rs (hunk 1 above)
Comment (ksayid) [reply src/tools/genpolicy/src/containerd.rs]: done!

**Review by SethHollandsworth — APPROVED** [review]
lgtm, we'll clean up later once we have a PoC

**Pull request merged**